peef/peef.go

```diff
@@ -91,3 +96,33 @@ var (
 		},
 	}
 )
+
+func getSymbolCurrentPrice(symbol string, key string) float32 {
+	var prices []Price
+
+	url := fmt.Sprintf("https://financialmodelingprep.com/api/v3/quote-short/%s?apikey=%s", symbol, key)
+	fmt.Println(url)
```

Comment:
Can use a debug log here instead of printing

Reply:
True but do you stream this to your console?

Reply:
Didn't notice `log.Info()` is set to do that. Noice.